Tasks/Xcode/xcodeutils.ts

```diff
@@ -46,6 +46,52 @@ export function findDeveloperDir(xcodeVersion: string): string {
     return discoveredDeveloperDir;
 }
 
+export function buildDestinationArgs(platform: string, devices: string[], targetingSimulators: boolean): string[] {
+    let destinations: string[] = [];
+
+    devices.forEach((device: string) => {
+        device = device.trim();
+
+        let destination;
+        if (device) {
+            if (targetingSimulators) {
+                destination = `platform=${platform} Simulator`;
+            }
+            else {
+                destination = `platform=${platform}`;
+            }
+
+            let matches = device.match(/([0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{12})/);
```

Comment:
Requires captials?

Reply:
I switched the case on a UUID and xcode rejected it. Probably better for me to be case-insensitive and still pass id=<input> to xcode though.